### PR TITLE
[NEW] Expand Apps Engine's environment variable allowed list

### PR DESCRIPTION
**Pull request by cuonghuunguyen**
## Proposed changes (including videos or screenshots)

Allows apps to read their own environment variables with the env var names start with `RC_APPS_${appID}`
## Issue(s)
Closes RocketChat/Rocket.Chat.Apps-engine#457

### CODE REVIEW

**Review by graywolf336 — DISMISSED** [review]
I like this idea and concept. While it might not be used by most people, especially Marketplace Apps, it can be insanely useful for private apps.

File: app/apps/server/bridges/environmental.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ export class AppEnvironmentalVariableBridge extends EnvironmentalVariableBridge
 	protected async isReadable(envVarName: string, appId: string): Promise<boolean> {
 		this.orch.debugLog(`The App ${appId} is checking if the environmental variable is readable ${envVarName}.`);
 
-		return this.allowed.includes(envVarName.toUpperCase());
+		return this.allowed.includes(envVarName.toUpperCase()) || envVarName.toUpperCase().startsWith(`RC_APPS_${ appId }_`);
```
Comment (debdutdeb) [inline app/apps/server/bridges/environmental.ts:27]:
```suggestion
		return this.allowed.includes(envVarName.toUpperCase()) || envVarName.toUpperCase().startsWith(`RC_APPS_${appId}_`);
```

**Review by debdutdeb — DISMISSED** [review]

**Review by d-gubert — APPROVED** [review]
Looking good, thanks!

**Comment by debdutdeb** [comment]
Ah this is gonna be wonderful!

**Comment by graywolf336** [comment]
Might add a comment or documentation that the id has to be all uppercase in the environmental

**Pull request merged**